app/elements/element-action-menu/element-action-menu.html

```diff
@@ -47,7 +47,10 @@
     _element: Object,
     iconsOnly: {type: Boolean, value: false, reflectToAttribute: true}
   },
-  githubLink: function(source) {
+  repositoryLink: function(source) {
+    return (source.indexOf('http') >= 0) ? source : this._githubLink(source);
```

Comment:
Currently it will match if `http` is in any part of the source url.  Can you change it to make sure it matches only if it starts with `http`?